streaming/base/format/xsv/encodings.py

```diff
@@ -94,28 +100,28 @@ def is_xsv_encoding(encoding: str) -> bool:
 
 
 def xsv_encode(encoding: str, value: Any) -> str:
-    """Encode the object.
+    """Encode the given data from the original object to string.
 
     Args:
-        encoding (str): The encoding.
-        value (Any): The object.
+        encoding (str): Encoding.
+        value (Any): Decoded object.
```

Comment:
Encoded

Reply:
*Object to encode

Reply:
Done in https://github.com/mosaicml/streaming/pull/72/commits/f202a4ff6281c757fc5cc6f0d11e3e2edfbef424.